funnel/models/event.py

```diff
@@ -28,7 +28,30 @@ def make_private_key():
     )
 
 
-class Event(BaseScopedNameMixin, db.Model):
+class ScopedNameTitleMixin(object):
+    # TODO: Move this into coaster?
+    @classmethod
+    def get(cls, parent, current_name=None, current_title=None):
+        if current_name:
```

Comment:
This shouldn't be a problem for `Event` and `TicketType` because of their uniqueness constraints. But, from a more general perspective, yeah, there should be a case that checks for both `current_title` and `current_name` being passed.